src/Uno.UI.Runtime.Skia.Gtk/GLESRenderSurface.cs

```diff
@@ -0,0 +1,197 @@
+﻿#nullable enable
+
+using System;
+using System.IO;
+using SkiaSharp;
+using Uno.Extensions;
+using Uno.UI.Xaml.Core;
+using Windows.UI.Xaml.Input;
+using WUX = Windows.UI.Xaml;
+using Uno.Foundation.Logging;
+using Windows.UI.Xaml.Controls;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Uno.UI.Runtime.Skia.Helpers.Windows;
+using Uno.UI.Runtime.Skia.Helpers.Dpi;
+using Windows.Graphics.Display;
+using Gdk;
+using System.Reflection;
+using Gtk;
+using Silk.NET.OpenGLES;
+using Silk.NET.Core.Loader;
+
+namespace Uno.UI.Runtime.Skia
+{
+
+	internal class GLESRenderSurface : GLArea, IRenderSurface
+	{
+		private const SKColorType colorType = SKColorType.Rgba8888;
```

Comment:
> Testing here I also noticed that the colors are kind of weird, the red is blue 🥲 ...

This looks like the surface is `Bgra8888` (and not `Rgba8888`). See https://github.com/unoplatform/uno/pull/8591 for a similar case